collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
@zouyee @cofyc Thanks for the info!

Since [`GetPersistentVolumeClaimClass`](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/helper/helpers.go#L561) is an exportable function, reference that function is more preferable instead of copying in another place. 

This makes kube-state-metrics more maintainable. 

WDYT @cofyc .